Source/WTF/wtf/RetainPtr.h

```diff
@@ -65,9 +65,30 @@ typedef struct objc_object *id;
 // both ways, we need a separate copy of our code when ARC is enabled.
 #if __has_feature(objc_arc)
 #define adoptNS adoptNSArc
+#define retainFoundationPtr retainFoundationPtrArc
+#define releaseFoundationPtr releaseFoundationPtrArc
+#define autoreleaseFoundationPtr autoreleaseFoundationPtrArc
 #define RetainPtr RetainPtrArc
 #endif
 
+inline void retainFoundationPtr(CFTypeRef ptr) { CFRetain(ptr); }
+inline void releaseFoundationPtr(CFTypeRef ptr) { CFRelease(ptr); }
+#if HAVE(CFAUTORELEASE)
+inline void autoreleaseFoundationPtr(CFTypeRef ptr) { CFAutorelease(ptr); }
+#endif
```

Comment:
Nit: Separate from this patch, I think we can probably remove `HAVE(CFAUTORELEASE)` because it only existed for the Apple Windows port.